docs/main/updating/5-0.md

```diff
@@ -10,7 +10,23 @@ Compared to previous upgrades, the breaking changes between Capacitor 4 and 5 ar
 
 ## Using the CLI to Migrate
 
-Install the latest version of the Capacitor CLI to your project using `npm i -D @capacitor/cli@latest`. Once installed, simply run `npx cap migrate` to have the CLI handle the migration for you. If any of the steps for the migration are not able to be completed, additional information will be made available in the output in the terminal. The steps for doing the migration manually are listed out below.
+Install the latest version of the Capacitor CLI to your project:
+
+```sh
+npm i -D @capacitor/cli@latest
```

Comment:
```suggestion
npm i -D @capacitor/cli@next
```
since it's still beta, it has to be installed with next tag, not with latest, I've seen several users confused about this